src/hoc/DataChanges.tsx

```diff
@@ -514,7 +516,8 @@ export function DataChanges(props: PropsWithChildren) {
   const saving = useMemo(() => anySaving(), [toolsChanged]);
   const doSanityCheck = useSanityCheck(setLanguage);
   const { getBibleMediaProject, getBibleMediaPlan } = useBibleMedia();
-  const fetchUrl = useFetchUrlNow();
+  const ts: ISharedStrings = useSelector(sharedSelector, shallowEqual);
+  const fetchUrl = useFetchUrlNow(ts.expiredToken);
```

Comment:
If we didn't pass in the expired token value, we wouldn't need any changes to this module.